__tests__/realtime-performance-optimization.test.ts

```
@@ -0,0 +1,620 @@
+  untrack: jest.fn().mockResolvedValue({ status: 'ok' })
+};
+
+jest.mock('../lib/supabase', () => ({
+  default: {
+    channel: jest.fn(() => mockSupabaseChannel),
+    from: jest.fn(() => ({
+      select: jest.fn().mockReturnThis(),
+      eq: jest.fn().mockReturnThis(),
+      order: jest.fn().mockReturnThis(),
+      limit: jest.fn().mockReturnThis(),
+      then: jest.fn().mockResolvedValue({ data: [], error: null })
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)